cmd/starlark/starlark.go

```diff
@@ -24,6 +24,7 @@ import (
 var (
 	cpuprofile = flag.String("cpuprofile", "", "gather CPU profile in this file")
 	showenv    = flag.Bool("showenv", false, "on success, print final global environment")
+	execprog   = flag.String("c", "", "execute program `c`")
```

Comment:
`c` (in the usage) should be `stmt` or `prog`, perhaps?

Reply:
Changed to `prog`. It can be more than one stmt--it can contain semicolons.